app/src/main/res/layout/recent_patient_item_view.xml

```diff
@@ -6,7 +6,8 @@
   android:layout_height="wrap_content"
   android:layout_marginBottom="@dimen/spacing_8"
   android:layout_marginEnd="@dimen/spacing_8"
-  android:layout_marginStart="@dimen/spacing_8">
+  android:layout_marginStart="@dimen/spacing_8"
+  android:foreground="?selectableItemBackground">
```

Comment:
We aren't using ripples as selectors for cards in the app. Can you check what other layouts are using?

Reply:
Fixed